front/src/Data.elm

```diff
@@ -1385,7 +1401,7 @@ recipes =
             , { material = cognac, quantity = CL 1 }
             , { material = angosturaBitters, quantity = Dash 2 }
             , { material = sugar, quantity = Cube 1 }
-            , { material = tripleSec, quantity = FewDrops } -- Optional
+            , { material = grandMarnier, quantity = FewDrops } -- Optional
```

Comment:
👍 after looking into it, these un-triple-sec'ing decisions look legit - this recipe still refers to triple sec though, so should probably refer to Grand Marnier instead.